### PR TITLE
Fix hardhat local devnet

### PR DESCRIPTION
**Pull request by boyuan-chen**
The new contract `SuperchainConfig` is added to the deployment pipeline. The `SuperchainConfig` can pause the bridges.

How to test it:
```
make devnet-hardhat-up
make devnet-hardhat-test
```

### CODE REVIEW

**Review by jyellick — APPROVED** [review]
Ran this locally, works for me!

**Review by mmontour1306 — APPROVED** [review]
Withdrawals are not working on my local system, but if it's working for others then I'm OK approving it rather than delaying other work while I debug it.

File: packages/contracts-bedrock/deploy/015-OptimismMintableERC20FactoryImpl.ts [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
 const deployFn: DeployFunction = async (hre) => {
+  const L1StandardBridgeProxy = await getContractFromArtifact(
+    hre,
+    'Proxy__OVM_L1StandardBridge'
+  )
   await deploy({
     hre,
     name: 'OptimismMintableERC20Factory',
-    args: [],
+    args: [L1StandardBridgeProxy.address],
```
Comment (souradeep-das) [inline packages/contracts-bedrock/deploy/015-OptimismMintableERC20FactoryImpl.ts:18]: this should be the L2StandardBridge?

**Review by souradeep-das — COMMENTED** [review]
lgtm! tests also pass for me. 
added one comment, please address it before merge

File: packages/contracts-bedrock/deploy/015-OptimismMintableERC20FactoryImpl.ts (hunk 1 above)
Comment (boyuan-chen) [reply packages/contracts-bedrock/deploy/015-OptimismMintableERC20FactoryImpl.ts:18]: No, this is deployed on L1.
https://github.com/bobanetwork/v3-anchorage/blob/8269ff508f6d92481d6e67af0397b352a1cd4ba2/packages/contracts-bedrock/scripts/Deploy.s.sol#L557-L574

Comment (souradeep-das) [reply packages/contracts-bedrock/deploy/015-OptimismMintableERC20FactoryImpl.ts:18]: ok thanks I see, the one for L2 is a predeploy.

**Review by souradeep-das — APPROVED** [review]

**Pull request merged**